kubernetes/create_cluster/script.md

```diff
@@ -75,7 +75,7 @@ credentials. We can specific them in the command if we want to (see
 minutes.
 
 ```
-az acs create --orchestrator-type=kubernetes --resource-group=$ACS_RESOURCE_GROUP --name=$ACS_CLUSTER_NAME --generate-ssh-keys
+az acs create --orchestrator-type=kubernetes --resource-group=$ACS_RESOURCE_GROUP --name=$ACS_CLUSTER_NAME --generate-ssh-keys --service-principal ${SERVICE_PRINCIPAL_ID} --client-secret ${SERVICE_PRINCIPAL_SECRET_KEY}
```

Comment:
The goal is to have the CI solution work with *any* script. This approach is problematic because it requires the scripts to be written in a specific way.

`az acs create` does not require an SP, it works perfectly well if the `az login` was executed earlier.